### PR TITLE
PR To Main from local to include RPI4 into IoT-Plug-And-Play

### PR DESCRIPTION
**Pull request by fmendoz7**
### Thank you for contributing to the Azure IoT Plug and Play Models repository

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

The repository pipeline will ensure minimum validation of incoming DTDL models.  

- PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).
- The same validation can be run locally via the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet#device-model-repository-client) CLI.

## Requested info template for model(s) submission

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve Plug and Play.

:star2: Please replace the markdown comment examples with your own values.

### Company Info

Raspberry Pi
https://www.raspberrypi.org/

### PnP Device Info

https://www.raspberrypi.org/products/raspberry-pi-4-model-b/?resellerType=home
Linux, Ubuntu 20.04 LTS
ARM64 Processor Architecture
128000000 kilobytes total storage
4000000 kilobytes total memory

### Model Submission Goals
 
Presence in certified device catalog (https://devicecatalog.azure.com/)
Custom Solution

### Code Owners & Reserved Names

@fmendoz7

### CODE REVIEW

**Review by rido-min — CHANGES_REQUESTED** [review]
The interface is using a reserved word in the id, as described [here](https://github.com/Azure/iot-plugandplay-models/blob/main/pr-reqs.md#reserved-words)

Also, this interface seems the same as DeviceInfo.

Is this submission associated with the raspberrypi team?

**Comment by fmendoz7** [comment]
Hi. This submission is not associated with the RP team, rather, this is a project under Intel Corporation wanting to use RPI4 in the Azure device marketplace for a project, but not able to find such model

**Review by rido-min — CHANGES_REQUESTED** [review]
There are still some issues in this PR.
- The `@id` used should not includes reserved words (as azure)
- I'd suggest to use more explicit ids than `dtmi:rpi4`
- There are unwanted files in the PR ( in the `,vs/` folder )

**Comment by fmendoz7** [comment]
Apologies, but I am a little confused as to how I am failing the validate-dtdl-models check. The @id for rpi4-1.json is "dtmi:raspberrypi:rpi4;1", but the presence of the reserved keyword "azure" is not on that file

File: dtmi/azure/devicemanagement/dtmi/azure/devicemanagement/rpi4-1.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azure:DeviceManagement:rpi4;1",
```
Comment (rido-min) [inline dtmi/azure/devicemanagement/dtmi/azure/devicemanagement/rpi4-1.json:3]: dont use `dtmi:azure` prefix

**Comment by rido-min** [comment]
I see the `azure` prefix in `dtmi/azure/devicemanagement/dtmi/azure/devicemanagement/rpi4-1.json`

Please note there is also an issue in the path (seems dtmi/ is dupe).

I'd suggest to use the `dmr-client` to validate locally before publishing, and to copy the files to the right location.
https://github.com/Azure/iot-plugandplay-models#install-dmr-client

**Comment by rido-min** [comment]
@fmendoz7 do you have any plans to update this PR? if not, can you please close it?

**Pull request closed without merging**